.gitignore

```diff
@@ -82,3 +82,5 @@ app.*.map.json
 !**/ios/**/default.perspectivev3
 !/packages/flutter_tools/test/data/dart_dependencies_test/**/.packages
 
+# Login credentials
+lib/services/credentials.dart
```

Comment:
I suggest using `lib/api_credentials.dart` or `lib/api_secure.dart` for consistency, since `api_keys.dart` is also stored in that location and serves a similar function

Reply:
Agreed.